src/components/Urlsnarf/Urlsnarf.tsx

```diff
@@ -7,21 +7,35 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "Urlsnarf";
+const title = "URLSnarf";
 const description_userguide =
-    "Urlsnarf is a network traffic sniffing tool that works to output all URL's that are requested from HTTP " +
```

Comment:
Please change "URL's" to "URLs" because it is plural not possessive. 

- [ ] Update grammatical error